code/src/components/TodoSummary.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { todos } from "reducers/todos";
+import styled from "styled-components";
+import swal from "sweetalert";
+import { Button } from "lib/Button";
+
+export const TodoSummary = () => {
+  const list = useSelector((store) => store.todos.list);
+  const doneList = list.items.filter((item) => item.done);
```

Comment:
if you only need the length of the done list, save just that info here.
`const doneList = list.items.filter((item) => item.done).length`